testonly/hammer/hammer.go

```diff
@@ -491,37 +491,30 @@ func (s *hammerState) retryOneOp(ctx context.Context) (err error) {
 	return firstErr
 }
 
-func (s *hammerState) performOp(ctx context.Context, ep MapEntrypointName, prng *rand.Rand) error {
-	switch ep {
-	case GetLeavesName:
-		return s.validReadOps.getLeaves(ctx, prng)
-	case GetLeavesRevName:
-		return s.validReadOps.getLeavesRev(ctx, prng)
-	case GetSMRName:
-		return s.validReadOps.getSMR(ctx, prng)
-	case GetSMRRevName:
-		return s.validReadOps.getSMRRev(ctx, prng)
-	case SetLeavesName:
-		return s.setLeaves(ctx, prng)
-	default:
-		return fmt.Errorf("internal error: unknown entrypoint %s selected for valid request", ep)
-	}
+type readOps interface {
+	getLeaves(context.Context, *rand.Rand) error
+	getLeavesRev(context.Context, *rand.Rand) error
+	getSMR(context.Context, *rand.Rand) error
+	getSMRRev(context.Context, *rand.Rand) error
 }
 
-func (s *hammerState) performInvalidOp(ctx context.Context, ep MapEntrypointName, prng *rand.Rand) error {
+type setLeaves func(context.Context, *rand.Rand) error
```

Comment:
nit: `setLeavesFn` or smth, to make it not look like an actual function name.

Reply:
Done.